site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -259,6 +259,9 @@ The following tables lists the configurable parameters of the Agones chart and t
 | Parameter                             | Description                                                                                                                                                                                                                           | Default |
 |---------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
 |                                                          |                                                                                                                                                                                                                                                            |
+| `agones.controller.replicas`                                   | The number of replicas to run in the deployment                                                                                                                                                                         | `2`                                |
+| `agones.controller.pdb.minAvailable`        | Description of the number of pods from that set that must still be available after the eviction, even in the absence of the evicted pod. Can be either an absolute number or a percentage. Mutually Exclusive with `maxUnavailable`   | `1`     |
```

Comment:
Suggest adding same parenthetical and link as above: (Ignored unless `SplitControllerAndExtensions` enabled.), here and below.